### PR TITLE
[Audio] Fix local track documentation.

### LINKED ISSUE #4777: Confusing help files for playing a local track
# Other bugs

#### What were you trying to do?

I wanted to find out the command to play a local track.

#### What were you expecting to happen?

I expected to [p]help, [p]help local, [p]help play or [p]help local play to inform me about the correct command.

#### What actually happened?

[p]help local and [p]help local play make it look like [p]local play <parent folder>/track.mp3 would be the correct command. [p]help play doesn't include the information that this works for local files as well. It only mentions URLs, which are I understood to be links to the Internet, not inside Red's local folder (after all, a full file URL doesn't work, either).

#### How can we reproduce this issue?

Use the commands
[p]help
[p]help local
[p]help local play
[p]help play
and read the help given.

### PR DESCRIPTION
**Pull request by FayeDel**
## Type

- [ ] Bugfix
- [x] Enhancement
- [ ] New feature

### Description of the changes

This is a workaround to #4777. In particular, this rewrites the help documentation to be more useful to queue local tracks.

- While using `[p]`help local play, this denotes that to play a local track file, and not necessarily by album, to redirect them to `[p]` play.
- While using `[p]` help play, this denotes both regular play usage (URL, search query) and local track files query, whilst also referencing the issue's usage of correctly playing them.

### CODE REVIEW

**Comment by aikaterna** [comment]
Hey there, I had a moment to go over this and wanted to provide a bit of feedback.
>If there's no album folders, and instead a regular .mp3 file, use the play command instead of
>local play. For more help, check the help documentation for the play command.

I feel like this is false as you absolutely can use the local play command with tracks and that examples here would be more apt for the users for help. Files should not be placed in the root localtracks directory, and while your example of `[p]play localtracks/root_dir_file.mp3` works, this was not intended.

I am putting a placeholder in these docstring examples as `placeholder` as I have asked another member of the community to generate docs for Audio so that the appropriate link can be placed in the brackets in the examples. The link will include the text of the `?localtracks` cc in the Red support server. If you are in that server, https://discord.com/channels/133049272517001216/387398816317440000/813823539094224897 is the content.

For example, for the `local play` docstring I would prefer something of this nature, and you're welcome to copy these and use them in your PR:

"""Play a local track.

To set up your localtracks folders, please see the documentation at: `placeholder`  
To play a local track, either use the menu to choose a track or enter in the localtracks path directly with the play command.
To play an entire local folder, use `[p]help local folder` for instrctions.

Examples:
`[p]local play`
Open a menu to pick a file

`[p]play localtracks/album_folder/song_name.mp3`
OR
`[p]play album_folder/song_name.mp3`
Use a direct link relative to the localtracks folder
"""


An addition could also be made to the [p]local folder command docstring because of this situation. My suggestion:

"""Play all songs in a localtracks folder.

To set up your localtracks folders, please see the documentation at: `placeholder` 

Examples:
`[p]local folder`
Open a menu to pick a folder to queue

`[p]local folder folder_name`
Queues all of the tracks inside the folder_name folder
"""

In the play command docstring, I feel like you can be more explicit about the command usage you were suggesting. The localtracks path is hidden from users except for the bot owner as the full path should not be revealed to users.

"""Play a URL or search for a track.

To play a local track/file, the query should be `<parentfolder>/<filename>`.
If you are the bot owner, use `[p]audioset info` to display your localtracks path.
"""

Also, for your information, Audio documentation is being added in #4872 and I have asked Bobloy to include the localtracks information for the `placeholder`, I will add another comment here when we have that link finalized. I realize this means that your addition in this PR may not be added until #4872 is accepted and merged (so that we have a final URL for you here) but I intend to put some hands on the Audio docs review as well to get that settled soon(tm).

**Comment by Jackenmen** [comment]
Hello there @DeltaXWizard, do you still plan to work on this? There's no rush of course, just would be good to know :)

**Comment by FayeDel** [comment]
Oh hey! Sorry I was in the midst of uni and some other personal stuff (and I'm currently all over the place), but yes I'll work on it this weekend :)

File: redbot/cogs/audio/core/commands/localtracks.py
Comment (Drapersniper) [inline redbot/cogs/audio/core/commands/localtracks.py]: Given that the known majority of users are on windows, please use windows paths here.

File: redbot/cogs/audio/core/commands/localtracks.py
Comment (Drapersniper) [inline redbot/cogs/audio/core/commands/localtracks.py]: Given that the known majority of users are on windows, please use windows paths here.

File: redbot/cogs/audio/core/commands/player.py
Comment (Drapersniper) [inline redbot/cogs/audio/core/commands/player.py]: Given that the known majority of users are on windows, please use windows paths here.

File: redbot/cogs/audio/core/commands/player.py
Comment (Drapersniper) [inline redbot/cogs/audio/core/commands/player.py]: given that 99.99% of the users using this wont be the bot owner; please remove this.

File: redbot/cogs/audio/core/commands/localtracks.py
Comment (Drapersniper) [inline redbot/cogs/audio/core/commands/localtracks.py]: There is no documentation, please remove this.

**Review by Drapersniper — CHANGES_REQUESTED** [review]

File: redbot/cogs/audio/core/commands/player.py
Comment (Drapersniper) [inline redbot/cogs/audio/core/commands/player.py]:
```suggestion
                msg = _("Connection to Lavalink has failed")
```
This is the title, to keep the UI for audio consistent there's a few rules to follow.

Title messages shouldn't have ponctuation if there is a description, given that the description here is conditional on whos running it, now im questioning this suggestion ... ill let @aikaterna chime in but I would lean toward not having this dot here.

Comment (aikaterna) [reply redbot/cogs/audio/core/commands/player.py]: Sorry to be late on replying, no period is fine here because of what Draper said - there's a description so the "Connection to Lavalink has failed" is more of a title instead of a sentence.

**Review by Drapersniper — DISMISSED** [review]
Thanks for the contribution Delta!, I've made some tweaks to bring it in line with `[p]playlist` docstrings but nothing too extensive.

Approving this

**Review by Drapersniper — APPROVED** [review]

**Pull request merged**
